HW1/P4/P4 bfs.py

```diff
@@ -0,0 +1,50 @@
+#I discussed the solution for this problem with Avery Faller.
```

Comment:
Be careful having a space (' ') in your filename.  This can lead to filename errors doing I/O.  Generally, best practice is to use underscores or something